constants.config.js

```diff
@@ -298,24 +298,24 @@ var constants = {
             }
         },
         LANGUAGES : [
-            { value : "de_DE", name : "Deutsch" },
-            { value : "en_GB", name : "English" },
-            { value : "en_CA", name : "English (Canadian)" },
-            { value : "en_US", name : "English (United States)" },
-            { value : "en_AU", name : "English (Australia)" },
-            { value : "en_ASIA", name : "English (Asia)" },
-            { value : "en_SG", name : "English (Singapore)" },
-            { value : "es_ES", name : "Español" },
-            { value : "fr_FR", name : "Français" },
-            { value : "fr_CA", name : "Français (Canadien)" },
-            { value : "it_IT", name : "Italiano" },
-            { value : "lt_LT", name : "Lietuviškai" },
-            { value : "nl_NL", name : "Nederlands" },
-            { value : "pl_PL", name : "Polski" },
-            { value : "pt_PT", name : "Português" },
-            { value : "sk_SK", name : "Slovakian" },
-            { value : "fi_FI", name : "Suomi" },
-            { value : "cs_CZ", name : "Česky" }
+            { value : "de_DE", name : "Deutsch", default: false, fallback: false },
```

Comment:
Wouldn't it be easier to just add a new constant default_language:xx_XX, fallback_language:xx_XX? It is unnecessary to specify true/false for each language

Reply:
Also, why is the fallback sometimes *fr*, sometimes *en* ?

Reply:
depending of the target:
* for EU, default is fr_FR, fallback is the same (like before)
* for CA, default is fr_CA, fallback to fr_FR
* for US, default is en_US, fallback to en_GB